apps/system/js/bluetooth_transfer.js

```diff
@@ -643,6 +645,33 @@ var BluetoothTransfer = {
     var body = {id: 'unknownMediaTypeToOpenFile', args: {fileName: fileName}};
     CustomDialog.show('cannotOpenFile', body, confirm, null, screen)
     .setAttribute('data-z-index-level', 'system-dialog');
-  }
+  },
 
+  _cleanupTempFileInDir: function bt__cleanupTempFileInDir(evt) {
+    // Try to get the file from bluetooth temp folder.
+    var filePath = '.bluetooth/tmp/' + evt.fileName;
+    var storageType = 'sdcard';
+    var storage = navigator.getDeviceStorage(storageType);
```

Comment:
during a long file transfer, the user might have time to go to the settings app and change the default sdcard on devices that have more than one. And if the do that, then I don't think this code will properly delete the file.  Is the actual blob stored in the transferInfo object? If so, you could get the full path from that and not have to re-create the path here.